tests/rpc/test_rpc_apiserver.py

```diff
@@ -700,12 +700,45 @@ def test_api_edge_disabled(botclient, mocker, ticker, fee, markets):
     assert rc.json() == {"error": "Error querying /api/v1/edge: Edge is not enabled."}
 
 
-@pytest.mark.parametrize('is_short', [
-    (True),
-    (False),
-    (None),
-])
-def test_api_profit(botclient, mocker, ticker, fee, markets, is_short):
+@pytest.mark.parametrize(
+        'is_short, best_pair, best_rate, profit_all_coin, profit_all_fiat,'
```

Comment:
let me know if i should remove the space, looks more readable like this :/

Reply:
mmh, on the risk of making it more verbose than it already is - i'd probably pass it in as dict (which i hope makes it more readable)

so you only got 2 parameters - `is_short,expected` with `{'best_pair: 1234, <...> }` as 2nd parameter, - and within the test, you can use `expected['best_pair']` for the check

While it'll be more verbose - i think this is very prone to break (some formatting tools will realign the lines - so this will become a "let's count the fields" problem :O

Reply:
Ya, if not we can seperate the test... Currently is becoming count the field problem. 
Ok I think i will prefer a dict anyday :D